rinex/src/antex/record.rs

```diff
@@ -203,25 +412,25 @@ impl Merge for Record {
         Ok(lhs)
     }
     /// Merges `rhs` into `Self`
-    fn merge_mut(&mut self, rhs: &Self) -> Result<(), merge::Error> {
-        for antenna in rhs.iter() {
-            if self.contains(antenna) {
-                let (antenna, frequencies) = antenna;
-                for (aantenna, ffrequencies) in self.iter_mut() {
-                    if antenna == aantenna {
-                        // for this antenna
-                        // add missing frequencies
-                        for frequency in frequencies {
-                            if !ffrequencies.contains(frequency) {
-                                ffrequencies.push(frequency.clone());
-                            }
-                        }
-                    }
-                }
-            } else {
-                self.push(antenna.clone());
-            }
-        }
+    fn merge_mut(&mut self, _rhs: &Self) -> Result<(), merge::Error> {
```

Comment:
Is this supposed to do nothing?

Reply:
hum, good catch once again. I forgot to re-enable this.
Reworking ATX requires to rework this method, I just forgot about it.
I will actually push one or two tests for this too.

FYI for each RINEX format, the idea is to support the Merge trait, to merge RINEX B into RINEX A, if their types matches. Basically, this is a fileops that was historically provided by teqc. 
It turns out to actually be mandatory in RINEX post processing and I use it all the time.
RINEX post processing requires putting toghethers entire datasets. One RINEX spans 24h of data. With Merge we can put a dataset together, or even cover 48h of data for example. A RINEX by itself is not enough for navigation, you need at least 3 or 4 of them.